server/models/user.js

```diff
@@ -167,5 +168,6 @@ User.shouldBeAnonymized = user => {
 }
 
 User.formatters = {
-  position: truncateLatLng
+  username: normalizeString,
```

Comment:
as moved to the model, i would also move related api tests to integration tests, to not overtest api which is testing a standard JS function (`.normalize`)

Reply:
the goal of the tests I wrote was not to test `.normalize` itself, but that `.normalize` was indeed called

Reply:
I dont understand why there are model formaters, if its to repeat the formatting in the controllers sanitizer.
I would be in favor of only formatting username in the user model (i dont see any inconvenient so far).